### PR TITLE
Solved underline issue in About page

### LINKED ISSUE #1258: Translation updates on hold
Hi, I've paused translation updates due to unresolved parsing issue I commented in https://github.com/commons-app/apps-android-commons/commit/96173e26cf07ef0d3f23cb8bb9ca4b21820eddf5#r27787891

Let's try to find a quick way to solve this.

### PR DESCRIPTION
**Pull request by Mansi449**
## Description

Fixes #1258 
Used CDATA to use HTML tags.

## Tests performed

Tested on Motorola Moto G(5) (Android 7.0 API 24)

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/AboutActivity.java [hunk 1]
```diff
@@ -62,6 +63,18 @@ public void onCreate(Bundle savedInstanceState) {
...
+        TextView translate = findViewById(R.id.about_translate);
+        TextView credits = findViewById(R.id.about_credits);
+        TextView faq = findViewById(R.id.about_faq);
+
+        rate_us.setText(Html.fromHtml(getString(R.string.about_rate_us)));
+        privacy_policy.setText(Html.fromHtml(getString(R.string.about_privacy_policy)));
+        translate.setText(Html.fromHtml(getString(R.string.about_translate)));
+        credits.setText(Html.fromHtml(getString(R.string.about_credits)));
+        faq.setText(Html.fromHtml(getString(R.string.about_faq)));
+
```
Comment (misaochan) [inline app/src/main/java/fr/free/nrw/commons/AboutActivity.java]: The CDATA changes look fine, but why the switch to TextViews?

Comment (Mansi449) [reply app/src/main/java/fr/free/nrw/commons/AboutActivity.java]: If the text is not set like this then it does not work as expected i.e. if we specify 
`<TextView
                android:text="@string/about_rate_us"
                />`

then the output is:
<u> Rate us </u>

Comment (misaochan) [reply app/src/main/java/fr/free/nrw/commons/AboutActivity.java]: Ah, okay, looks good in that case. :)

**Comment by misaochan** [comment]
Tested, looks good to me. Thanks @Mansi449 !

**Pull request merged**